src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -152,7 +154,12 @@ def save
         begin
           authorize @package, :update?
 
-          Package.verify_file!(@package, '_patchinfo', xml)
+          begin
+            Package.verify_file!(@package, '_patchinfo', xml)
+          rescue APIException => e
```

Comment:
Hehe so immediately you forgot that you have to have code to handle the exception somewhere else. I guess you should have listened in the first place 😜